README.md

```diff
@@ -219,5 +220,22 @@ function toggleHidden(element) {
 	}
 }
 ```
+[↑ back to Table of Contents](#table-of-contents)
+
+## Use `.first()` for single element
+
+### Why?
+jQuery always returns a collection when using `$(selector)` or `$root.find(selector)`, while sometimes you are only interested in / only expect one element. In vanilla JS you would use `.querySelector(selector)` instead of `.querySelectorAll(selector)`. For clarity you should use [`.first()`](http://api.jquery.com/first/).
+
+### How?
+
+```javascript
+// collection of buttons:
+$buttons = $form.find('button');
```

Comment:
.first() returns a jQuery object. Whether you use .get() or nor later I think does not make **this** rule it confusing.